docs/design/normalization/custom.md

```diff
@@ -181,7 +181,9 @@ Possible combinations and their encoding:
 
 _The rows of the table, from bottom to top, are encoded with increasing 16-bit "norm16" values as noted in the last column. Per-row and per-row-group properties are determined via norm16 range checks._
 
-<table border="1" bordercolor="#888888" style="border-color:rgb(136,136,136);border-width:1px;border-collapse:collapse;border-spacing:0" data-table-local-id="table-2">
+<style type="text/css">.compacttable th,.compacttable td { padding: unset; min-width: unset; }</style>
```

Comment:
Add a comment mentioning that the default styling is applying `padding` and `max-width` to `th` and `td` in a way that greatly bloats the tables.

Reply:
done